test/unit/selector.js

```diff
@@ -1326,3 +1326,19 @@ test("select() with pre-compiled function", function() {
 		equal( Sizzle.select( compiled, Sizzle( "#first")[0] ).length, 0, "Should not match with different context" );
 	});
 });
+
+test("Sizzle supports shadow DOM nodes as root", function() {
+	if ( !document.documentElement.createShadowRoot ) {
+		ok( true, "Shadow DOM not supported in this browser" );
+		return;
```

Comment:
We should also stop using the globals as we've done in jQuery. `QUnit.test` instead of `test`, `assert` as a test function parameter etc. Perhaps @markelog could apply one of those magic scripts that he used for jQuery. :-)